secp256k1-sys/Cargo.toml

```diff
@@ -19,7 +19,7 @@ links = "rustsecp256k1_v0_1_1"
 features = [ "recovery", "endomorphism", "lowmemory" ]
 
 [build-dependencies]
-cc = ">= 1.0.28, <= 1.0.41"
+cc = "1.0.28"
```

Comment:
This is confusing - if you intend it to be 1.0.X can you not write 1.0?

Reply:
this was `>= 1.0.28` previously and I assumed this was for a reason, so I didn't change that

Reply:
and `1.0.28` means `^1.0.28`, not `=1.0.28` https://doc.rust-lang.org/cargo/reference/specifying-dependencies.html#specifying-dependencies-from-cratesio

Reply:
Wait, so we still depend on exactly cc 1.0.28? I thought the point was that we wanted to bump it to a newer version?

Reply:
`Cargo.toml` specifies a range of compatible versions, not a particular version. The latter is specified in `Cargo.toml`. `1.0.28` means `^1.0.28` which is `>= 1.0.28, < 2.0.0`.

Reply:
Ah, ok. Wow that is mega-confusing, `^` implies a regex to me...

Reply:
Yes. I mislead @TheBlueMatt, as @ordian tried to say `cc = "1.0.28` means that the version is bigger or equal to `1.0.28` but less than `2.0.0`

Reply:
Well, that's called semver :)